robot/Cumulus/resources/NPSP.py

```diff
@@ -1512,6 +1593,7 @@ def click_show_more_actions_button(self,title):
         locator=npsp_lex_locators['link-contains'].format("more actions")
         self.selenium.wait_until_element_is_visible(locator)
         self.selenium.click_element(locator)
+        time.sleep(1)
```

Comment:
same as mentioned above you can just use self.builtin.sleep(1)  which is part of Robot.